src/components/signup/SignupState.js

```diff
@@ -177,7 +181,7 @@ const Signup = ({ navigation }: { navigation: any, screenProps: any }) => {
   //so wallet can use it, if torus is enabled and we dont have pkey then require re-login
   const checkTorusLogin = () => {
     const masterSeed = torusUserFromProps.privateKey
-    if (Config.torusEnabled && masterSeed === undefined) {
+    if (!Config.enableSelfCustody && Config.torusEnabled && masterSeed === undefined) {
```

Comment:
All places that rely on Config.torusEnabled should actually rely on the signin method the user actually started.
meaning from the Auth screen we need to pass the registration method 'torus'/'selfcustody' to signupstate
we skip some steps if Config.torusEnabled

Reply:
done